data/versions.json

```diff
@@ -0,0 +1,62 @@
+{
+    "current": {
+        "name": "v7.0",
+        "date_start": "July 8, 2020",
+        "changelog": "updates-july-2020",
+        "cti_url": "https://github.com/mitre/cti/releases/tag/ATT%26CK-v7.0"
+    },
+    "previous": [
+        {
+            "name": "v6.3",
+            "aliases": [],
+            "date_start": "October 24, 2019",
+            "date_end": "July 7, 2020",
+            "changelog": "updates-october-2019",
+            "cti_url": "https://github.com/mitre/cti/releases/tag/ATT%26CK-v6.3",
+            "commit": "b15db5ce0b2f440cb690a3003ffc689098625d00"
```

Comment:
This is actually a TODO for me, but we need to change all Navigator links to use the old Navigator. [I still need to stand up an instance](https://github.com/mitre-attack/attack-navigator/issues/143) but then I'll branch of gh-pages and do a global replace for the navigator link to target the preserved previous version.

Reply:
To be clear, this applies to the v6.3 version of the website, not the v7.0 version.